src/components/Table/utils/RowState.ts

```diff
@@ -111,14 +113,13 @@ export class RowState {
    * child of a selected parent row.
    */
   select(selected: boolean): void {
-    if (this.row.selectable === false) return;
```

Comment:
Removed this early return so that unselectable rows will still ignore the "set my selected yes/no", but now will also push the new selected state down to their children.

I.e. the kept row is not selectable, but we want the header being selected/unselected to "push through it" to the children kept rows.